apps/docs/pages/guides/database/tables.mdx

```diff
@@ -242,8 +242,15 @@ You would [connect](../../guides/database/connecting-to-postgres#direct-connecti
 
 ```bash
 psql -h DATABASE_URL -p 5432 -d postgres -U postgres \
-  -c "COPY movies FROM './movies.csv';"
+  -c "\COPY movies FROM './movies.csv';"
 ```
+Additionally use the `DELIMITER`, `HEADER` and `FORMAT` options as defined in the PostgreSQL [COPY](https://www.postgresql.org/docs/current/sql-copy.html) docs.
+```bash
+psql -h DATABASE_URL -p 5432 -d postgres -U postgres \
+  -c "\COPY movies FROM './movies.csv' WITH DELIMITER ',' CSV HEADER"
+```
+
+Tip: If you get a `FATAL:  password authentication failed for user "postgres"` error, reset the password in your Database settings and try again.
```

Comment:
```suggestion
  -c "\COPY movies FROM './movies.csv';"
```

Additionally use the `DELIMITER`, `HEADER` and `FORMAT` options as defined in the PostgreSQL [COPY](https://www.postgresql.org/docs/current/sql-copy.html) docs.

```bash
psql -h DATABASE_URL -p 5432 -d postgres -U postgres \
  -c "\COPY movies FROM './movies.csv' WITH DELIMITER ',' CSV HEADER"
```

Tip: If you get a `FATAL:  password authentication failed for user "postgres"` error, reset the password in your Database settings and try again.
```

Reply:
Ah sorry - this suggestion-commit clobbered your update. I'll fix it